### DISCUSSION TITLE
Presto driver accidentally deleted

### DISCUSSION THREAD

**Discussion by tripathirishabh8** [dbeaver/dbeaver#8700] [Q&A] [answered]
Hi,

I am newbie at dBeaver and accidentally deleted Presto from Driver Manager.
I tried reinstalling dbeaver yet it is not coming back.
How to get it back?

Thanks 
Rishabh

**Comment by serge-rider** [comment, marked as answer]
It is in the driver manager dialog:
![image](https://user-images.githubusercontent.com/6398845/81958421-a7fc5400-9616-11ea-9488-7d10c66fccd6.png)

**Comment by tripathirishabh8** [comment]
Hi serge-rider,

I am on version 6 32-bit windows and I dont have undelete option.
Any other step can you please provide?

Thanks 
Rishabh

**Comment by serge-rider** [comment]
Unfortunately we don't support 32-bit version anymore.

You could try to edit/delete drivers.xml file manually.
https://github.com/dbeaver/dbeaver/wiki/Admin-Manage-Drivers
